packages/data/lib/dio/dio_builder.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:data/assets/constants.dart';
+import 'package:dio/dio.dart';
+import 'package:injectable/injectable.dart';
+
+@module
+abstract class DioBuilder {
+  Dio dioBuilder() {
+    final baseOptional = BaseOptions(baseUrl: baseUrl);
+    final client = Dio(baseOptional);
+    client.interceptors.add(
+      LogInterceptor(
+          request: false, responseHeader: false, requestHeader: false,),
```

Comment:
formatting - press command + option + L if u r working in studio. in vscode - idk

Reply:
fixed